test/acsengine.deploy

```diff
@@ -101,6 +105,12 @@ uksouth ukwest"
             // Generate reports
             try {
               junit("${junit_dir}/**/junit.xml")
+              if(sendReport && (sendTo != "")) {
+                emailext(
+                  to: "${sendTo}",
+                  subject: "[ACS Engine Jenkins Failure] ${env.JOB_NAME} #${env.BUILD_NUMBER}",
```

Comment:
*nit:* Maybe add something that tells what kind of test from a quick glance since I assume other types of jobs will be extended to send mail as well... Is it a smoke failure, a BVT failure?